src/app/components/client/SubscriberWaitlistDialog.tsx

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import {
+  ReactElement,
+  RefObject,
+  cloneElement,
+  forwardRef,
+  useRef,
+} from "react";
+import Image from "next/image";
+import { OverlayTriggerAria, useOverlayTrigger } from "react-aria";
+import { OverlayTriggerState, useOverlayTriggerState } from "react-stately";
+import { Button } from "./Button";
+import { Dialog } from "./dialog/Dialog";
+import { ModalOverlay } from "./dialog/ModalOverlay";
+import ModalImage from "../client/assets/subscriber-waitlist-dialog-icon.svg";
+import { useL10n } from "../../hooks/l10n";
+import styles from "./SubscriberWaitlistDialog.module.scss";
+
+function WaitlistDialog({
+  dialogTriggerState,
+  overlayTrigger,
+}: {
+  dialogTriggerState: OverlayTriggerState;
+  overlayTrigger: OverlayTriggerAria;
+}) {
+  const l10n = useL10n();
+
+  return (
+    dialogTriggerState.isOpen && (
+      <ModalOverlay
+        {...overlayTrigger.overlayProps}
+        state={dialogTriggerState}
+        isDismissable={true}
+      >
+        <Dialog
+          title={l10n.getString("subscriber-waitlist-dialog-title")}
+          illustration={<Image src={ModalImage} alt="" />}
+          onDismiss={() => dialogTriggerState.close()}
+        >
+          <div className={styles.dialogContent}>
+            {l10n.getString("subscriber-waitlist-dialog-info-text")}
+            <br />
+            <br />
+            {l10n.getString("subscriber-waitlist-dialog-instruction-text")}
+
+            <div className={styles.buttonWrapper}>
+              <Button
+                variant="primary"
+                href={process.env.NEXT_PUBLIC_WAITLIST_URL}
+              >
+                {l10n.getString("subscriber-waitlist-dialog-cta-button-label")}
+              </Button>
+              <Button
+                variant="secondary"
```

Comment:
Isn't this also supposed to be a `tertiary` button? (And stacked vertically?)

Reply:
Updated in [13889ee](https://github.com/mozilla/blurts-server/pull/3918/commits/13889eed6907886e4dbf5f91061d87df8b77c6d9).